eng/Build.props

```diff
@@ -12,8 +12,8 @@
       to the source-build package cache when building with source-build to allow them to be
       considered in prebuilt reporting.
 
-      All newly added packages should be grouped together at the end of the list. The reason this is 
-      important is that when previous source-built artifacts are updated, the entries for the new projects 
```

Comment:
Ooops, these were unintentional, my editor removes trailing spaces, do you want me to revert them?